### PR TITLE
SW-19025 Added: Documentation for our Grunt features in Shopware 5.3

### PR DESCRIPTION
**Pull request by klarstil**

### CODE REVIEW

File: source/designers-guide/best-practice-theme-development/index.md [hunk 1]
```diff
@@ -92,3 +92,57 @@ grunt --shopId 1 # optionally specify shopId
...
+### Modularized Grunt tasks
+The next big improve are the modularized Grunt tasks. We're using auto-loading for Grunt plugins and a JIT (Just in Time) plugin loader which allows us to separate the different tasks and configurations into separate files. 
+
+![Grunt Screenshot](grunt-modularized.png)
+
+The advantage of this approach is that you as a third party developer are able to register your own tasks and add new plugins like a FTP deployment for example without changing the core `Gruntfile.js`.
+
+
+#### Adding your own tasks
+Adding a new task is easy now. You just have to install the Grunt plugin you want using `npm / yarn` and create a new configuration for the tasks in the `grunt-tasks/config` directory. Please keep in my the file has to be named along the task name. For example if you're installing the plugin [`grunt-contrib-clean`](https://github.com/gruntjs/grunt-contrib-clean) you create a new file `clean.js` in the `config` folder.
```
Comment (PaddyS) [inline source/designers-guide/best-practice-theme-development/index.md]: Please keep in *mind ?
Or what did you try to say here?

Comment (klarstil) [inline source/designers-guide/best-practice-theme-development/index.md]: Yup, that's what I was trying to say, good catch 👍

File: source/designers-guide/best-practice-theme-development/index.md [hunk 2]
```diff
@@ -92,3 +92,57 @@ grunt --shopId 1 # optionally specify shopId
...
+
+```
+module.exports = {
+    build: {
+        src: ['path/to/dir/one', 'path/to/dir/two']
+    }
+};
+```
+
+Now you just have it's possible to use the task using:
```
Comment (PaddyS) [inline source/designers-guide/best-practice-theme-development/index.md]: *Now it's possible to run the task using:

File: source/designers-guide/best-practice-theme-development/index.md (hunk 1 above)
Comment (klarstil) [inline source/designers-guide/best-practice-theme-development/index.md]: Done!

File: source/designers-guide/best-practice-theme-development/index.md (hunk 2 above)
Comment (klarstil) [inline source/designers-guide/best-practice-theme-development/index.md]: Done!

File: source/designers-guide/best-practice-theme-development/index.md [hunk 3]
```diff
@@ -92,3 +92,57 @@ grunt --shopId 1 # optionally specify shopId
 ```
 
 ![Grunt Screenshot](grunt-screenshot.png)
+
+## New in Shopware 5.3 - LiveReload & modularized grunt tasks
+
+In Shopware 5.3 we added a couple new features to our Grunt installation. First of all we've added a LiveReload mode which automatically reloads your browser when the Grunt compilation is successful which should speed up your workflow quite a bit.
```
Comment (hype09) [inline source/designers-guide/best-practice-theme-development/index.md]: a couple of*

**Pull request merged**